x-pack/solutions/observability/plugins/slo/public/pages/slos/components/health_callout/health_callout.tsx

```diff
@@ -68,12 +58,13 @@ export function HealthCallout({ sloList }: { sloList: SLOWithSummaryResponse[] }
       title={
         <FormattedMessage
           id="xpack.slo.sloList.healthCallout.title"
-          defaultMessage="Transform error detected"
+          defaultMessage="Some SLOs are unhealthy"
```

Comment:
Do we need to remove the translations if we modify this message? So that they can get recreated.